docs/DeveloperGuide.md

```diff
@@ -332,33 +332,77 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  User requests to add contact
-2.  User inputs the contact details
-3.  User requests to add the contact
-4.  Medicontact adds the contact
+1. User requests to add contact.
+2. User inputs the contact details.
+3. MediContact adds the contact.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. Necessary field is missing
+* 2a. Necessary field is missing.
+    * 2a1. MediContact shows an error message indicating which field is missing.
+  
+      Use case ends.
 
-    * 2a1. Medicontact shows an error message indicating which field is missing
-    * 2a2. Use case ends.
+* 2b. Wrong format in input.
+    * 2b1.MediContact shows an error message specifying the incorrect format.
+  
+      Use case ends.
 
-* 2b. Wrong format in input
+**Use case: Delete a person**
+
+**MSS**
 
-    * 2b1.Medicontact shows an error message specifying the incorrect format.
-    * 2b2. Use case ends.
+1.  User requests to list persons. 
+2.  MediContact shows a list of persons. 
+3.  User requests to delete a specific person in the list. 
+4.  MediContact requests for confirmation. 
+5.  User confirms to delete. 
+6.  MediContact deletes the person.
 
-**Use case: Delete a person**
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index or name is invalid.
+  - 3a1. AddressBook shows an error message.
+  
+    Use case resumes at step 2.
+  
+* 4a. User chooses to cancel the deletion.
+
+  Use case ends.
+
+**Use case: Clear address book**
```

Comment:
I think "Clear all contacts in MediContact" might be a better way to phrase this.